documentation/docs/welcome.md

```diff
@@ -1,36 +1,36 @@
 ---
+description: Introducing Hornet, a powerful community driven IOTA node software written in GO.
+image: /img/logo/HornetLogo.png
 keywords:
 - IOTA Node 
 - Hornet Node
 - Hornet
 - IOTA
 - Node Software
 - Welcome
-description: Introducing Hornet, a powerful community driven IOTA node software written in GO.
-image: /img/logo/HornetLogo.png
+- explanation
 ---
 
 # Welcome to HORNET
 
 ![Hornet Logo](/img/logo/HORNET_logo.svg)
 
-HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end
-devices like the Raspberry Pi 4. a group of community developers alongside the
-IOTA Foundation are responsible for building and maintaining Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the
+HORNET is a powerful, community driven IOTA node software written in Go. It is easy to install and runs on low-end devices like the Raspberry Pi 4., a group of community developers alongside the IOTA Foundation are responsible for building and maintaining Hornet. Hornet is a full-fledged software providing full node capabilities including full support of the
```

Comment:
We should remove the Raspi 4 from all the docs.

Reply:
I think my last commit should take care of that.  Please review the hornet_apt_repository.md file